lib/elixir/lib/map.ex

```diff
@@ -88,8 +88,10 @@ defmodule Map do
       iex> map = %{one: 1, two: 2}
       iex> %{map | one: "one"}
       %{one: "one", two: 2}
-      iex> %{map | three: 3}
-      ** (KeyError) key :three not found
+
+  When a key is updated that does not exist on the map a key error will be raised:
```

Comment:
It is not as easy to grasp as this one
```suggestion
  When a key that does not exist in the map is updated a key error will be raised:
```